src/Microsoft.AspNet.Mvc.TagHelpers/TagHelperContentExtensions.cs

```diff
@@ -51,7 +52,18 @@ public static class TagHelperContentExtensions
 
```

Comment:
That said, this class is getting uglier and the Razor changes may provide a way to get rid of it. Just not sure whether rewriting those two tag helpers should be done in this PR versus just after.